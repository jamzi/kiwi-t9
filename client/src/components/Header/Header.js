import React from 'react';
import AppBar from '@material-ui/core/AppBar';
import Toolbar from '@material-ui/core/Toolbar';
import Typography from '@material-ui/core/Typography';

const Header = () => (
  <AppBar position="static" color="primary">
    <Toolbar>
      <Typography variant="title" color="inherit">
        Kiwi T9
      </Typography>
    </Toolbar>
  </AppBar>
);

export default Header;
