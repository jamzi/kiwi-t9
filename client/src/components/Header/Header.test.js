import React from 'react';
import ReactDOM from 'react-dom';
import Header from './Header';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
