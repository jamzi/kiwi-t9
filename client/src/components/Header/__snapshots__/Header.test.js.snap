// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header
  className="MuiPaper-root-9 MuiPaper-elevation4-15 MuiAppBar-root-1 MuiAppBar-positionStatic-5 MuiAppBar-colorPrimary-7"
>
  <div
    className="MuiToolbar-root-36 MuiToolbar-gutters-37"
  >
    <h2
      className="MuiTypography-root-38 MuiTypography-title-44 MuiTypography-colorInherit-57"
    >
      Kiwi T9
    </h2>
  </div>
</header>
`;
